library/common/main_interface.cc

```diff
@@ -56,6 +71,8 @@ envoy_status_t run_engine(envoy_engine_t, envoy_engine_callbacks callbacks, cons
                           const char* log_level) {
   // This will change once multiple engine support is in place.
   // https://github.com/lyft/envoy-mobile/issues/332
-  engine_ = std::make_unique<Envoy::Engine>(callbacks, config, log_level, preferred_network_);
+  static auto internal =
```

Comment:
Not a big deal, but for consistency of our statics, should we declare this above as internal_engine_ or strong_engine_, or better_name_?

Reply:
Definitely we can change the name, but I did think it made more sense, at present, to encapsulate its declaration in this function call:
- nothing else should ever touch it, and this limits its accessible scope, and
- its lifecycle is more deterministic here.

Reply:
Thanks for the explanation. I think the first point is sufficient on its own to leave it here.